rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -3075,7 +3044,28 @@ error_code sys_fs_get_mount_info_size(ppu_thread&, vm::ptr<u64> len)
 		return CELL_EFAULT;
 	}
 
-	*len = lv2_fs_object::get_mount_count();
+	*len = 0;
```

Comment:
Why?

Reply:
Though it was created by me in #12917, there is no apparent necessity for get_mount_count() to be separated from sys_fs_get_mount_info_size(), which is the only syscall that uses it.